content/account/users.textile

```diff
@@ -72,6 +72,32 @@ To remove a user from your account:
 3. Click the *Delete* button next to the user to remove from the account.
 4. Confirm the action.
 
-h2(#delete). Delete your account
-
-Follow "this process":https://faqs.ably.com/how-do-i-cancel-my-account to delete your Ably account. Only an account owner can initiate this process.
+h2(#close). Close your account
+
+You can close your account at any time by downgrading to the "Free package":/docs/pricing/free. The following steps cancel your account:
+
+* Log in to your "account":https://ably.com/accounts/any.
+* Ensure you are the account "owner":/docs/account/users#roles.
+* Downgrade your paid "package":/docs/pricing#packages to Free:
+** Navigate to *Account* then "Billing":https://ably.com/accounts/any/package.
+** Click *Save billing details* to save your changes.
+* After your account has been downgraded to *Free*, go to the "My Settings":https://ably.com/users/edit page.
+** Navigate to *Want to delete your profile?*.
+** Click *start* to begin the closure process.
```

Comment:
The closure process will only be available on the next billing cycle(first of the month) as we ensure that the account has no outstanding invoices

Reply:
done 👉 ff291a8acb1c489c95e5b4bc1a1ab7047b03e973